exercise3.js

```diff
@@ -86,50 +114,106 @@
 const numbers = [2,23,1,2,1,1,1,2,2.5,20,200,2000,,{k:1},20000,19999,1878,140,23,4,"sk",true,true,"true-dat","nice","one","two","three","3","tea",[]];
 
 function maxNumber(numbers) {
-    //your code...
-}
+  //your code...
+  const truthys = numbers.filter(el => el);
+  const primitives = truthys.filter(n => typeof n != 'object');
+  const noBools = primitives.filter(p => typeof p != 'boolean');
+  const numStrings = ["zero","one","two","three"];
+  const numWithStrings = noBools.map(nb => numStrings.includes(nb) ? numStrings.indexOf(nb) : nb);
+  const numWithStrings2 = numWithStrings;
+  numWithStrings2[24] = Number(numWithStrings[24]);
+  const nums = numWithStrings2.filter(n => typeof n != 'string');
+
+  return Math.max(...nums);
+};
 
 // 7.b -Write a function that sorts the given numbers array.  Allow the function to sort the array in descending order
 
 function sortNums(numbers,desc=false) {
-    //your code...
+  //your code...
+  return numbers.sort().reverse();
 };
```

Comment:
you should provide the option to sort in ascending `default` or descending order, depending on the boolean flag for the 2nd parameter.  you are only doing descending here.